### PR TITLE
Added the ability to specify the pull request comment as a template.

### PR DESCRIPTION
**Pull request by VladMasarik**
Added the ability to specify the pull request comment as an environment variable. 
It is possible to reference the environment variables inside the comment using in similar manner: 
`TEMPLATE="The benchmark is starting. Your Github token is {{ index . \"GITHUB_TOKEN\" }}."`

### CODE REVIEW

File: cmd/tools/commentMonitor/main.go [hunk 1]
```diff
@@ -66,27 +66,25 @@ func main() {
...
+		envvar := make(map[string]string)
+		for _, e := range env {
+			tmp := strings.Split(e, "=")
+			envvar[tmp[0]] = tmp[1]
+		}
 
-The Prometheus servers being benchmarked can be viewed at :
-- PR - [prombench.prometheus.io/%v/prometheus-pr](%v/%v/prometheus-pr)
-- %v - [prombench.prometheus.io/%v/prometheus-release](%v/%v/prometheus-release)
+		unformatedComment := os.Getenv("TEMPLATE_BENCH")
```
Comment (krasi-georgiev) [inline cmd/tools/commentMonitor/main.go]: this needs to be passed as an arg as well.


```
template := app.Arg("template", "Template text so use when posting a github comment. All env variables will be replaces with.......").Required().String()

var unformatedComment string
unformatedComment = os.Getenv(template)
if unformatedComment ==""{
	unformatedComment = os.ReadFile(template)
}

if unformatedComment == "" ...fail
```

File: cmd/tools/commentMonitor/main.go [hunk 2]
```diff
@@ -132,6 +130,10 @@ func writeArgs(args []string, output string) {
 		if err != nil {
 			log.Fatalln(err)
 		}
+		err = os.Setenv(filename, string(data))
```
Comment (krasi-georgiev) [inline cmd/tools/commentMonitor/main.go:140]: why is this needed?

**Review by krasi-georgiev — COMMENTED** [review]
just a bit if clarification for @nikita-mk .

this change is required so that the same tool can be used running benchmark tests for tsdb. The comment for this test is different so we need a way to set this dynamically. 

After this change the template text would need to be set as an env variable.

**Comment by krasi-georgiev** [comment]
Maybe also add an example template to give an idea of how it would look like.

File: cmd/tools/commentMonitor/main.go (hunk 2 above)
Comment (VladMasarik) [reply cmd/tools/commentMonitor/main.go:140]: Since we want to use env vars in the template and `args` such as PR number and branch are not env vars, I had to export those arguments so that I can use them in the comment.

File: cmd/tools/commentMonitor/main.go (hunk 1 above)
Comment (VladMasarik) [reply cmd/tools/commentMonitor/main.go]: Does it have to be an argument or is `flag` enough?

**Comment by VladMasarik** [comment]
> Maybe also add an example template to give an idea of how it would look like.

I hoped that the example in the starting comment is enough, but I added a larger example.

File: cmd/tools/commentMonitor/main.go (hunk 1 above)
Comment (krasi-georgiev) [reply cmd/tools/commentMonitor/main.go]: yes flag is better.

**Comment by krasi-georgiev** [comment]
> I hoped that the example in the starting comment is enough, but I added a larger example.


I meant somewhere in main.go so that when people do ... -h it shows an example template to start with.

File: cmd/tools/commentMonitor/main.go (hunk 2 above)
Comment (krasi-georgiev) [reply cmd/tools/commentMonitor/main.go:140]: ack. Could you add a comment like
```
// Also add all comment args as env variables so that these can be used in the comment template.
```

File: cmd/tools/commentMonitor/main.go [hunk 3]
```diff
@@ -66,27 +67,25 @@ func main() {
...
+		envvar := make(map[string]string)
+		for _, e := range env {
+			tmp := strings.Split(e, "=")
+			envvar[tmp[0]] = tmp[1]
+		}
 
-The Prometheus servers being benchmarked can be viewed at :
-- PR - [prombench.prometheus.io/%v/prometheus-pr](%v/%v/prometheus-pr)
-- %v - [prombench.prometheus.io/%v/prometheus-release](%v/%v/prometheus-release)
+		unformatedComment := os.Getenv(templateEnvVar)
```
Comment (krasi-georgiev) [inline cmd/tools/commentMonitor/main.go]:
```suggestion
		unformatedComment := os.Getenv(*templateEnvVar)
```

File: cmd/tools/commentMonitor/main.go (hunk 2 above)
Comment (VladMasarik) [reply cmd/tools/commentMonitor/main.go:140]: Added the comment.

**Comment by VladMasarik** [comment]
I renamed some variables because only now I noticed that I used the ones when I needed a quick variable name.

**Comment by krasi-georgiev** [comment]
ok, ping me when ready for a final review after adding the skip-member-validation and posting a comments when the validation fails.

**Comment by krasi-georgiev** [comment]
good work, Thanks!

**Comment by krasi-georgiev** [comment]
@nikita-mk 	now you need to set a new env var that would include the template text.

**Comment by nikita-mk** [comment]
I am working on it.
